### PR TITLE
Docs staging

### PR DESCRIPTION
**Pull request by fadeev**
Updated to the latest version of the theme.

`<span v-pre>` is added to the following files to prevent docs build errors.

https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-051-double-signing-risk-reduction.md

https://github.com/tendermint/tendermint/blob/master/docs/architecture/adr-052-tendermint-mode.md

Also removed links to Riot.

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -19,12 +19,6 @@ module.exports = {
     gutter: {
       title: "Help & Support",
       editLink: true,
-      chat: {
```
Comment (marbar3778) [inline docs/.vuepress/config.js]: can we put in the discord instead of removing this?

File: docs/.vuepress/config.js [hunk 2]
```diff
@@ -95,10 +90,6 @@ module.exports = {
             {
               title: "Forum",
               url: "https://forum.cosmos.network/c/tendermint"
-            },
-            {
-              title: "Chat",
```
Comment (marbar3778) [inline docs/.vuepress/config.js]: we should have some chat, this is quite bad that we are getting rid of a way for people to talk to us

**Comment by marbar3778** [comment]
the fix to this would be when this issue is closed: https://github.com/vuejs/vuepress/issues/1558

**Comment by marbar3778** [comment]
would like to have some chat and not remove it entirely. we are closing off a gateway into the ecosystem by doing

File: docs/.vuepress/config.js (hunk 2 above)
Comment (fadeev) [reply docs/.vuepress/config.js]: Added link to Discord: https://discordapp.com/channels/669268347736686612.

**Pull request merged**